app-frontend/src/app/components/filterPane/filterPane.html

```diff
@@ -16,10 +16,30 @@
       <div class="filter-item">
         <div id="year_slider"></div>
       </div>
-      <rzslider rz-slider-model="$ctrl.yearFilters.minModel"
-                rz-slider-high="$ctrl.yearFilters.maxModel"
-                rz-slider-options="$ctrl.yearFilters.options"
-      ></rzslider>
+      <div class="btn-group">
+        <button type="button"
+                class="btn btn-toggle btn-small"
+                ng-class="{'active': $ctrl.yearFilterMode === 'range'}"
+                ng-click="$ctrl.setYearFilterMode('range')">
```

Comment:
Amazingly, I never hit the buttons more than once. Good catch! Will skip the logic if the mode remains the same.